lib/cc/cli/path_fingerprint.rb

```diff
@@ -0,0 +1,33 @@
+require "digest/md5"
+
+module CC
+  module CLI
+    class PathFingerprint
+      def initialize(output)
+        @output = output
+      end
+
+      def compute
+        md5 = Digest::MD5.new
+        md5 << key
+        md5.hexdigest
+      end
+
+      def key
+        "#{path}|#{check_name}"
```

Comment:
I think I'm leaning towards removing the separator entirely. I don't think it's very helpful. codeclimate-markdownlint doesn't [use one either](https://github.com/jpignata/codeclimate-markdownlint/blob/master/lib/cc/engine/markdownlint.rb#L93-L97).